app/recipes/routes.py

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/env python3
+import html
+import math
+import re
+from app import mongo
+from bson.objectid import ObjectId
+from datetime import datetime
+from flask import Blueprint, current_app, render_template, redirect, request, url_for, flash, session, Markup
+from slugify import slugify
+
+
+# --------------------- #
+#    Flask Blueprint    #
+# --------------------- #
+recipes = Blueprint("recipes", __name__)
+
+
+# -------------------- #
+#    DB Collections    #
+# -------------------- #
+allergens_collection = mongo.db.allergens
+desserts_collection = mongo.db.desserts
+measurements_collection = mongo.db.measurements
+recipes_collection = mongo.db.recipes
+users_collection = mongo.db.users
+
+
+# ------------------- #
+#    Global Helper    #
+# ------------------- #
+@recipes.context_processor
+def desserts_total():
+    desserts_count = recipes_collection.count
+    return dict(desserts_count=desserts_count)

```

Comment:
Since you're using this context processor on almost all blueprints, define the context_processor on the app object directly so you don't have to repeat the code on all blueprints